src/main/navigation/MainNav.svelte

```
@@ -1,69 +1,87 @@
 <script lang="ts">
-import { onMount } from 'svelte';
 import { cubicInOut } from 'svelte/easing';
-import { crossfade } from 'svelte/transition';
+import { crossfade, scale } from 'svelte/transition';
```

ESLint

[error] 'scale' is defined but never used.

(unused-imports/no-unused-imports)